frontend/src/app/core/hubs/issues-hub.service.ts

```diff
@@ -33,11 +34,13 @@ export class BroadcastHubService {
 
     private async init() {
         await this.hubConnection.start()
-            .then(() => console.info(`"${this.hubFactory}" successfully started.`))
-            .catch(() => console.info(`"${this.hubFactory}" failed.`));
+            .then(() => {
+                console.info('IssuesHub successfully started.');
+            })
+            .catch(() => console.info('IssuesHub start failed.'));
```

Comment:
Please use async/await instead .then and .catch

Reply:
Done